### PR TITLE
Test tweaks 

### PR DESCRIPTION
**Pull request by erp12**
Resolves issues 2 through 4 of #210.

Happy to make any adjustments that you would like!

### CODE REVIEW

File: test/zero_one/geni/rdd_test.clj [hunk 1]
```diff
@@ -53,7 +53,7 @@
     (rdd/master) => "local[*]"
     (rdd/persistent-rdds) => map?
     (rdd/resources) => {}
-    (rdd/spark-home) => nil?
+    (rdd/spark-home) => any?
```
Comment (anthony-khong) [inline test/zero_one/geni/rdd_test.clj]: Actually, I believe this should be checked against the real Spark home using `(System/getenv "SPARK_HOME")`. Apologies, I should've said on the original issue thread!

**Review by anthony-khong — COMMENTED** [review]
Thank you so much for this! This is awesome! I just have one tiny comment on `any?`. I hope that's fine!

**Comment by anthony-khong** [comment]
Hi Eddie, I believe the next time you run the CI pipeline, the `lint-ancient` ancient job will fail. You will have to upgrade the `lein-cloverage` to `1.2.1` in `project.clj`.

File: test/zero_one/geni/rdd_test.clj (hunk 1 above)
Comment (erp12) [reply test/zero_one/geni/rdd_test.clj]: Good point, that is much better. Thanks! Just fixed it.

**Comment by erp12** [comment]
@anthony-khong Thanks for the review. I have addressed the PR comment and rebased on master so that the `lein-cloverage` version bump is in the history. Let me know if there is anything else you would like me to do.

**Comment by anthony-khong** [comment]
Thanks again @erp12. This is awesome!

**Pull request merged**